source/game/system/KPadController.hh

```diff
@@ -150,6 +153,23 @@ private:
     bool m_acceptingInputs;
 };
 
+/// @brief The abstraction of a controller object but for external usage.
+/// @details The input state is managed externally by programs interfacing with Kinoko.
+class KPadHostController : public KPadController {
```

Comment:
Do any design decisions need to be figured out in order to make this class accessible via ABI?

Reply:
It can be accessed via the KPadDirector if necessary. However, my assumption is that most people are simply compiling Kinoko with their project and not linking as an external library, in which case it's expected they will have a pointer to this controller to manage its input state.